sw/host/cryptotest/testvectors/data/BUILD

```diff
@@ -105,6 +105,21 @@ genrule(
     ] + extra_msg_types
 ]
 
+genrule(
```

Comment:
Is there a reason for using genrule over run_binary here?

Reply:
Only because this was copied from the AES version. I'll fix it to use `run_binary`.

Reply:
Fixed now